### ISSUE TITLE
Redis Cluster: "Disconnect on error: Too many cluster redirects. Connection aborted." & app freeze

### ISSUE THREAD

**Issue by neohq** [redis/RedisDesktopManager#4987]
I've managed to open the log and saw this error but most of the time the app is just freezing and I need to close it.
The redis cluster is inside a Docker container.

**NOTE**: I've tested with other apps like [redis-commander](https://www.npmjs.com/package/redis-commander) and everything works well.

**Environment (please complete the following information):**
 - OS & version: Windows 10 20H2
 - Redis-Server version: 6.0.1
 - Docker version: 3.1.0 (51484)
 - RDM version: `rdm-2021.0.278.exe`

**Cluster info**

```
cluster_state:ok
cluster_slots_assigned:16384
cluster_slots_ok:16384
cluster_slots_pfail:0
cluster_slots_fail:0
cluster_known_nodes:6
cluster_size:3
cluster_current_epoch:6
cluster_my_epoch:1
cluster_stats_messages_ping_sent:335
cluster_stats_messages_pong_sent:331
cluster_stats_messages_sent:666
cluster_stats_messages_ping_received:326
cluster_stats_messages_pong_received:335
cluster_stats_messages_meet_received:5
cluster_stats_messages_received:666
```

**Cluster slots**

```
1) 1) "0"
   2) "5460"
   3) 1) "127.0.0.1"
      2) "7001"
      3) "aad201dfcac4b41331cde4451a789821f4c7e7d3"

   4) 1) "127.0.0.1"
      2) "7006"
      3) "4b5a847cf6a653e31aa3f09dcdb160d7ae6b857b"


2) 1) "10923"
   2) "16383"
   3) 1) "127.0.0.1"
      2) "7003"
      3) "f4caf7c586bb48d8b01992b1fa99a744f9d0f6b2"

   4) 1) "127.0.0.1"
      2) "7005"
      3) "ade417641260623d10424301b62bd2449347a720"


3) 1) "5461"
   2) "10922"
   3) 1) "127.0.0.1"
      2) "7002"
      3) "3009929a69a81874ee41a3065a1db84a4776e8f1"

   4) 1) "127.0.0.1"
      2) "7004"
      3) "b87f01238e77c295278661721e10f38d80319c2a"
```
![Screenshot_2553](https://user-images.githubusercontent.com/10179436/106791419-8daf4e00-665d-11eb-9854-d5d2d7508d0f.png)

**Comment by neohq** [comment]
Unfortunately the behaviour is the same on MacOS 11.2, only this time the app crashes and is closing itself.

**Comment by netwerkz** [comment]
@uglide why did you close this? I have the same problem.

**Comment by uglide** [comment]
@netwerkz This issue was closed automatically because fix was released.

The fix was released as part of 2020.2 release and is available on the website.

**Comment by netwerkz** [comment]
> @netwerkz This issue was closed automatically because fix was released.
> 
> The fix was released as part of 2020.2 release and is available on the website.

Thank you for fixing it. Appreciate it.
